### PR TITLE
Sample poi parallel

### PR DESCRIPTION
**Pull request by tzufgoogle**
1. Generate samples parallel to speed up generation. 
2. Added version to Wikigeo
3. Removed S2Graph creation as we don't use it.

### CODE REVIEW

File: cabby/geo/geo_item.py [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+      path_features,
+      main_pivot,
+      near_pivot, 
+      beyond_pivot
+    )
+
+  @classmethod
+  def empty(cls):
+    """Construct an Entity from the start and end points, route, and pivots.
+    """
```
Comment (jasonbaldridge) [inline cabby/geo/geo_item.py:65]: This doc string seems incorrect.

File: cabby/geo/sample_poi.py [hunk 2]
```diff
@@ -56,8 +56,8 @@ def main(argv):
     FLAGS.region, FLAGS.min_s2_level, FLAGS.directory)
```
Comment (jasonbaldridge) [inline cabby/geo/sample_poi.py:59]: Make sure to merged with master and update this to use the new region stuff that was committed a bit ago, e.g.:

```
map_region = map_structure.Map(
    regions.get_region(FLAGS.region), FLAGS.min_s2_level, FLAGS.directory)	 
```

File: cabby/rvs/item.py [hunk 3]
```diff
@@ -41,60 +41,59 @@ class RVSPath:
   `from_file` should the entity be processed from file.
   `instructions` is a basic template that includes the points, pivots and route 
   features.
+  `path_features` is a path features: cardinal directions, intersections, instructions and route geometry.
```
Comment (jasonbaldridge) [inline cabby/rvs/item.py:44]: >80 chars

File: cabby/rvs/item.py [hunk 4]
```diff
@@ -41,60 +41,59 @@ class RVSPath:
...
-    if not self.process:
-      return
-    # Creat basic template instruction.
-    if "main_tag" in self.beyond_pivot:
-      avoid_instruction = "If you reached {0}, you have gone too far.".format(
-        self.beyond_pivot['main_tag'])
 
+    if "main_tag" in self.beyond_pivot:
+      avoid_instruction = (
+        f"If you reached {self.beyond_pivot['main_tag']}, you have gone too far.")
```
Comment (jasonbaldridge) [inline cabby/rvs/item.py:62]: "if you reached" -> "if you reach"

**Review by jasonbaldridge — CHANGES_REQUESTED** [review]
Great! Just a few small things to fix up.

File: cabby/geo/geo_item.py (hunk 1 above)
Comment (tzufgoogle) [reply cabby/geo/geo_item.py]: Thanks, fixed.

File: cabby/geo/sample_poi.py (hunk 2 above)
Comment (tzufgoogle) [reply cabby/geo/sample_poi.py]: Already did :)

File: cabby/rvs/item.py (hunk 3 above)
Comment (tzufgoogle) [reply cabby/rvs/item.py]: fixed

File: cabby/rvs/item.py (hunk 4 above)
Comment (tzufgoogle) [reply cabby/rvs/item.py]: Fixed and made sure it is also correct in the real templates (it was).

**Review by tzufgoogle — COMMENTED** [review]
Changed the comments as suggested.

**Review by jasonbaldridge — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**